lib/muser_priv.h

```diff
@@ -35,6 +35,8 @@
 
 #include "muser.h"
 
+#define UNUSED __attribute__((unused))
```

Comment:
Why not just drop this define?

Reply:
This is the define which is used for unused variables everywhere internally.

Reply:
I know, why not use directly `__attribute__((unused))`?

Reply:
This way it looks better and readable to me. Also using __attribute__ all over needs extra indentation.